docs/source/editor.rst

```diff
@@ -1,35 +1,24 @@
 
-==============
-File Structure
-==============
+==================
+Editor syntax
+==================
 
-The structure of a ``btm`` file is the following:
 
-- *package* declaration (optional)
-- *network* declaration (optional)
-- *constants* and *transactions* declaration
-- *compile* statement (optional)
-
--------
-Package
--------
-
-The package declaration specify where the generated files must be saved. 
-By default, generated files are saved in ``src-gen`` with a directory structure reflecting the package name.
-For example, declaring :btm:`package com.example.test`, the generated files will be store
-in ``src-gen/com/example/test/``.
+"""""""""""
+Comments
+"""""""""""
+It is possible to add comments to the written  code. This functionality is more useful when using the local eclipse plugin, respect to the online version. 
```

Comment:
This functionality is more useful when using the local eclipse plugin, respect to the online version. 
Non l'ho capita

Reply:
ok